packages/react/src/components/Divider/Divider.mdx

```diff
@@ -6,10 +6,56 @@ import * as DividerStories from './Divider.stories';
 <Meta of={DividerStories} />
 
 # Divider
+`Divider` brukes for å skape et visuelt skille mellom innhold. Det er en enkel horisontal linje som strekker seg over tilgjengelig bredde.
 
 <Primary />
 <Controls />
 
+<br/>
+
+## Slik bruker du `Divider`
+```tsx
+<Divider />
+```
+
+<br/>
+
+## Retningslinjer for `Divider`
+
+`Divider` brukes for å dele opp innhold i mindre deler, slik at det blir mer oversiktlig og lettere å lese. Den kan også brukes for å skille innhold som er relatert til hverandre, men som likevel bør ha en visuelt skille.
+
+<br/>
+Passer:
+* Når du vil skape en tydelig oppdeling av innhold.
+* Når whitespace alene ikke gir tilstrekkelig separasjon.
+* Når du ønsker å strukturere innhold for bedre lesbarhet.
+
+<br/>
+
+Passer ikke:
+* Når naturlig spacing eller andre visuelle elementer gir tilstrekkelig separasjon.
+* Når for mange divider-elementer skaper et rotete visuelt uttrykk.
+
+<br/>
+
+## Tilgjengelighet
+
+* Sørg for at `divider` har tilstrekkelig kontrast mot bakgrunnen for god synlighet.
+* Unngå å bruke `divider` utelukkende for visuell effekt uten funksjonell betydning.
+
+<br/>
+
+### Kjente begrensninger
+
+* `Divider` støtter kun horisontal deling av innhold. Vertikale delere er ikke en del av denne komponenten.
+* Stilen til divider kan påvirkes av foreldreelementer, spesielt ved bruk i fleksible layouts.
```

Comment:
Dette kan forsåvidt bli fiksa av konsument, så usikker på om me burde skrive det her.
Komponenten baserer seg på `hr`, som står for "horizontal rule", altså ikkje vertikal.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr